scripts/tools.sh

```diff
@@ -11,8 +11,7 @@ cd "$SCRIPT_DIR"
 
 ./install-gotestsum.sh -b $(go env GOPATH)/bin
 ./install-goreleaser.sh -b $(go env GOPATH)/bin v0.182.1
-#curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.42.2
-go install github.com/golangci/golangci-lint/cmd/golangci-lint@master
-go install golang.org/x/tools/cmd/godoc@latest
+curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.50.1
```

Comment:
Stiahnutie binarky je ovela rychlejsie, mali sme to takto aj skor, ale bol tam nejaky problem a potrebovali sme `@master` verziu, ale uz tam problem nie je.